### ISSUE TITLE
How do I set the .env location and filename?

### ISSUE THREAD

**Issue by hugheba** [otkmnb2783/gradle-dotenv-plugin#1]
I'm trying to change the location and filename of the .env file but am unable to set it.

In my build.gradle I have
```
dotenv {
  dir = "../" 
  fileName = ".env.dev"
}
```

But when I run `gradle --debug someTask | grep dotenv` it's still looking for .env in the current folder.

Is that the correct syntax to override the dir and fileName?

**Comment by smac89** [comment]
I'd recommend using absolute paths. So something like `dir = project.rootDir.parentFile.absolutePath`.

You can test your configuration by creating some variables in `.env.dev`, then in your `build.gradle(.kts)`, add the following line:

```gradle
afterEvaluate {
    println project.env
}
```

Do you get the variables in the `.env.dev`?

**Comment by otkmnb2783** [comment]
Thanks for the report.

There was a bug where the extended object could not be retrieved properly.

This bug has been fixed in v0.2.0.
